daemon/daemon_test.go

```diff
@@ -41,11 +43,8 @@ func TestFailedReload(t *testing.T) {
 	}
 }
 
-var localServerPort = ":8081"
-var localServerHost = "http://localhost" + localServerPort
-var localProxyHost = "http://localhost:6634"
-
-func setUpDaemonWithLocalServer() error {
+func setUpDaemonWithLocalServer(localServerPort, localProxyListen, healthCheckPort string,
```

Comment:
Seems to me that this should just take a `config.Config`, since you pass in a lot of the variables and roll a `config.Config` with them?